book/src/fundamentals/zkbook_fft.md

```diff
@@ -1,8 +1,18 @@
 ### Multiplying polynomials with the Fast Fourier Transform (FFT)
 
 The algorithm that allows us to multiply polynomials in $O(n \log n)$ is called the **Cooley-Tukey fast Fourier transform**, or **FFT** for short.
+It was discovered by [Gauss](https://en.wikipedia.org/wiki/Carl_Friedrich_Gauss) 160 years earlier, but then separately rediscovered and publicized by Cooley-Tukey.
 
-The way this works is as follows. Say we have $\omega \in F$ such that $\omega^{n} = 1$, and $\omega^r \neq 1$ for any $0 < r < n$.
+The heart of Cooley-Tukey FFT is actually about converting between coefficient and evaluation representations, rather than the multiplication itself.
+Given polynomials $p$ and $q$ in dense coefficient representation, it works like this.
+1.  Convert $p$ and $q$ from coefficient to evaluation form in $O(n\log n)$ using Cooley-Tukey FFT
+2.  Multiply $p$ and $q$ in evaluation form by multiplying their points pairwise in $O(n)$
+3.  Convert $p$ and $q$ back to to coefficient form in $O(n\log n)$ using the inverse Cooley-Tukey FFT
```

Comment:
Shouldn't this be `Convert $p * q$ back to coefficient form`? Could even name the result of the multiplication `r` above, to make it clear.